views/default/elggx_userpoints/list.php

```diff
@@ -44,7 +44,7 @@
 		
 		// name
 		$row[] = elgg_format_element('td', ['width' => '50%'], elgg_view('output/url', [
-			'text' => $entity->getDisplayName(),
+			'text' => $entity->username,
```

Comment:
Why change this to username, and not keep the display name? If my username is `xwsdt` but by display name is `Jerome Bakker` i as an admin would prefer the display name. (or maybe `Jerome Bakker (xwsdt)`

Reply:
See also the answer below.

This list is on the same page as the restore input field where you need to enter the username. I thought I would keep it simple and only display the username on the whole page. As it's the admin backend I hadn't thought it would matter because no normal user would see that their username and not their display name is displayed (and if necessary worked with).

Showing both username (necessary to work with) and display name (for completeness) might be an option. Even more so as the details tab still uses the display name.